scripts/storage.js

```diff
@@ -0,0 +1,51 @@
+let state = {
+    currentQuestion: 0,
+    currentScore: 0,
+    currentQuiz: 0,
+
+}
+
+function setState(){
+    localStorage.setItem("currentState", JSON.stringify(state));
+}
+
+function getState(){
+    state = JSON.parse(localStorage.getItem("currentState"));
+}
+
+function addPoint(){
+    getState();
+    state.currentScore += 1;
+    setState();
+}
+
+function nextQuestion(){
+    getState();
+    state.currentQuestion += 1;
+    setState();
+}
+
+function getScore(){
+    getState();
+    return state.currentScore;
+}
+
+function getQuestion(){
+    getState();
+    return state.currentQuestion;

```

Comment:
That's wrong. You should return an object of question here. Like this 
```suggestion
   return questions[state.currentQuestion];
```

Reply:
Sure! May I make another function then, 
to get the index of the current question?